scheduler/src/cook/mesos/util.clj

```diff
@@ -438,3 +439,32 @@
   [instance]
   (some #{(:instance/status instance)} #{:instance.status/running
                                          :instance.status/unknown}))
+
+(defn close-when-ch!
+  "When the value passed in is a channel, close it. Otherwise, do nothing"
+  [maybe-ch]
+  (try
+    (async/close! maybe-ch)
+    (catch Exception e)))
+
+(defn chime-at-ch
```

Comment:
Wanna leave it for now? Thankfully, the isn't a public api so we can always rename later.